tasks/freebsd.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- name: install build dependencies
```

Comment:
looks like name is duplicated but task is different, could be good to change name. will check on #13